workspaces/liferay-testray-workspace/client-extensions/liferay-testray-custom-element/src/components/Form/Select/Select.tsx

```diff
@@ -35,6 +36,7 @@ const InputSelect: React.FC<InputSelectProps> = ({
 	name,
 	register = () => {},
 	id = name,
+	isLoading = true,
```

Comment:
```suggestion
	isLoading,
```
Since you're already setting the initial value of isLoading state as true, I believe you don't need to set the default value again here.